modules/services/aerospace/default.nix

```diff
@@ -71,9 +85,17 @@ in
               description = "Default orientation for the root container.";
             };
             on-window-detected = lib.mkOption {
-              type = listOf str;
+              type = listOf (attrsOf (either callbackConf (listOf callbackConf)));
```

Comment:
I think it would be better with `types.listOf (types.attrsOf types.anything)`

Ideally we would be able to type this better, but I'm not familiar with Aerospace

Reply:
I think this is cleaner indeed :) just pushed the changes